src/dfastbe/bank_erosion/data_models/inputs.py

```diff
@@ -267,6 +275,238 @@ def _read_river_axis(self) -> LineString:
+    reed: List[np.ndarray]
+
+    @classmethod
+    def get_ship_data(
+        cls, num_stations_per_bank: List[int], config_file: ConfigFile
+    ) -> "ShippingData":
+        """Get ship parameters from the configuration file.
+
+        Args:
+            num_stations_per_bank (List[int]):
+                The number of stations per bank.
+            config_file (ConfigFile):
+                Configuration file containing parameters.
+
+        Returns:
+            ShippingData: An instance of ShippingData with parameters read from the config file.
+        """
+
+        param_defs = cls._get_initial_parameter_definitions()
+        param_dict = {}
+
+        # Retrieve parameter values
+        for param in param_defs:
+            param_dict[f"{param.name.lower()}"] = cls._get_param(
+                config_file,
+                param.name,
+                num_stations_per_bank,
+                default=param.default,
+                valid=param.valid,
+                onefile=param.onefile,
+                positive=param.positive,
+                ext=param.ext,
+            )
+
+        return cls(config_file, **param_dict)
+
+    @staticmethod
+    def _get_param(
+        config_file: ConfigFile, name: str, num_stations_per_bank, iq_str="", **kwargs
+    ):
+        """Get a parameter from the configuration file.
+
+        Args:
+            config_file (ConfigFile):
+                Configuration file containing parameters.
+            name (str):
+                Name of the parameter to retrieve.
+            num_stations_per_bank (List[int]):
+                The number of stations per bank.
+            iq_str (str, optional):
+                String to append to the parameter name for specific levels.
+            **kwargs:
+                Additional keyword arguments for parameter retrieval.
+        Returns:
+            np.ndarray: The parameter value as a NumPy array.
+        """
+        return config_file.get_parameter(
+            "Erosion",
+            f"{name}{iq_str}",
+            num_stations_per_bank,
+            **kwargs,
+        )
+
+    @staticmethod
+    def _get_initial_parameter_definitions() -> List[Parameters]:
+        """Get parameter definitions for discharge parameters.
+
+        Args:
+            shipping_data (Dict[str, List[np.ndarray]]):
+                The shipping data.
+
+        Returns:
+            List[namedtuple]: List of parameter definitions.
+        """
+        return [
+            Parameters("VShip", None, None, True, True, None),
+            Parameters("NShip", None, None, True, True, None),
+            Parameters("NWave", 5, None, True, True, None),
+            Parameters("Draught", None, None, True, True, None),
+            Parameters("ShipType", None, [1, 2, 3], True, None, None),
+            Parameters("Slope", 20, None, None, True, "slp"),
+            Parameters("Reed", 0, None, None, True, "rdd"),
+        ]
+
+    @staticmethod
```

Comment:
why this is a static method when you actually need the values of the attributes below ?